.github/workflows/testing.yml

```diff
@@ -13,7 +13,6 @@ on:
 env:
   ACTIONS: 1
   LONG_TESTS: 0
-  nvd_api_key: ${{ secrets.NVD_API_KEY }}
```

Comment:
Can we add a comment here explaining that the nvd_api_key will be passed below to deal with a codecov jitter issue?  Just saves us forgetting and trying to optimize later. ;)